app/helpers/courses_helpers.rb

```diff
@@ -2,14 +2,13 @@
 module Sinatra
   module UMDIO
     module Helpers
-
       # Generic course helpers
 
       # generalize logic for checking if semester param valid
-      def check_semester app, semester
+      def check_semester(_app, semester)
```

Comment:
Doesn't look like `_app` is used, should it be removed entirely?

Reply:
I think so, I wasn't 100% comfortable removing parameters before understanding why they were there in the first place, but should be fine to remove